.chronus/changes/copilot-fix-7479-2025-4-28-19-54-3.md

```diff
@@ -0,0 +1,8 @@
+---
+# Change versionKind to one of: internal, fix, dependencies, feature, deprecation, breaking
+changeKind: internal
```

Comment:
This should be `fix` instead of `internal` so that the change is visible in the generated changelog.